Documentation/mender-ci-cd.md

```diff
@@ -67,19 +67,25 @@ demonstrate setting up the kernel server.
 
 Instead of listing all the configuration options for exports(5), the following 
 provides a brief example installing `nfs-kernel-server` and exporting the 
-directory `/nfs-dir` as read/write to all hosts on the sub-net `192.168.1.0/24`
+directory `/sstate-cache` as read/write to all hosts on the sub-net `192.168.1.0/24`
 and making it read-only for hosts outside this range.
 ```
 sudo apt-get install -y nfs-common nfs-kernel-server
-sudo mkdir -p /nfs-dir
-sudo chown -R 1000:1000 /nfs-dir # Set the owner 
+sudo mkdir -p /sstate-cache
+sudo chown -R 1000:1000 /sstate-cache # Set the owner
+
+# Mount the NFS drive
+sudo tee -a /etc/fstab > /dev/null << EOF
+UUID=3e4b3f08-6e97-464e-a03c-8e10124b3357   /sstate-cache    ext4    defaults    0    1
+EOF
+sudo mount -a
 
 # See exports(5) for detailed description of configuration options to /etc/exports
-sudo cat << EOF >> /etc/exports
-/nfs-dir 192.168.1.0/24(rw,insecure,all_squash,anonuid=1000,anongid=1000) *(ro,insecure,all_squash,anonuid=1000,anongid=1000)
+sudo tee -a /etc/exports > /dev/null << EOF
+/sstate-cache 10.162.0.0/32(rw,insecure,all_squash,anonuid=1010,anongid=1010) *(ro,insecure,all_squash,anonuid=1010,anongid=1010)
```

Comment:
Reminder: Let's talk about this on Monday.

Reply:
Your latest comments actually answered everything I wanted to know, so this is no longer necessary.